src/SubAPIMultiSig.sol

```diff
@@ -2,59 +2,50 @@
 
 pragma solidity 0.8.17;
 
-import "./MultiSig.sol";
+import {ECDSA} from "@openzeppelin/contracts@4.9.2/utils/cryptography/ECDSA.sol";
+import {OwnerManager} from "@safe-smart-account/base/OwnerManager.sol";
```

Comment:
import {OwnerManager} from "@safe-global/safe-contracts/contracts/base/OwnerManager.sol";

Reply:
we use the remapping. 
`@safe-smart-account/=lib/safe-smart-account/contracts/`